src/App.jsx

```diff
@@ -26,20 +27,15 @@ export const alex = {
 export const App = () => (
   <div className="App">
     <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
+      <Person person={misha} />
     </section>
 
     <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
+      <Person person={olya} />
     </section>
 
     <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
+      <Person person={alex} />
     </section>
```

Comment:
```
<div className="App">
    <Person person={misha} />
    <Person person={olya} />
    <Person person={alex} />
</div>
```
section should be in Person component

Reply:
But I haven't this in my code 
![image](https://user-images.githubusercontent.com/80322190/228435000-62a5c9ee-3ccd-4b92-b7b2-eb43650a3ba4.png)